activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -999,7 +999,7 @@ def build_from
       case opts
       when Relation
         name ||= 'subquery'
-        self.bind_values = opts.bind_values + self.bind_values
+        self.bind_values += opts.bind_values + opts.arel.bind_values
```

Comment:
This changes order between `self.bind_values` and `opts.bind_values`. Is it important?